x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields.tsx

```diff
@@ -92,38 +92,63 @@ const columns: Array<EuiBasicTableColumn<HighlightedFieldsTableRow>> = [
  * Component that displays the highlighted fields in the right panel under the Investigation section.
  */
 export const HighlightedFields: FC = () => {
-  const { dataFormattedForFieldBrowser, scopeId, isPreview } = useDocumentDetailsContext();
-  const { ruleId } = useBasicDataFromDetailsData(dataFormattedForFieldBrowser);
-  const { loading, rule: maybeRule } = useRuleWithFallback(ruleId);
+  const {
+    dataFormattedForFieldBrowser,
+    scopeId,
+    isPreview,
+    investigationFields,
+    rule,
+    isExistingRule,
+  } = useDocumentDetailsContext();
+  const [isEditLoading, setIsEditLoading] = useState(false);
+  const editHighlightedFieldsEnabled = useIsExperimentalFeatureEnabled(
+    'editHighlightedFieldsEnabled'
+  );
 
   const highlightedFields = useHighlightedFields({
     dataFormattedForFieldBrowser,
-    investigationFields: maybeRule?.investigation_fields?.field_names ?? [],
+    investigationFields,
+    type: 'all',
```

Comment:
Minor thing but you don't need `type: 'all'` here as it's the default value in your hook